lib/scripts/services/contextMenu.js

```diff
@@ -1,19 +1,57 @@
+function _checkIfWordCountNotOne(inputString){
```

Comment:
I use underscores in function names to easily differentiate between the helper/private methods and the class methods